src/sonic-yang-models/yang-models/sonic-system-radius.yang

```diff
@@ -0,0 +1,214 @@
+module sonic-system-radius {
+    namespace "http://github.com/sonic-net/sonic-system-radius";
+    prefix ssys;
+    yang-version 1.1;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-port {
+        prefix port;
+    }
+
+    import sonic-portchannel {
+        prefix lag;
+    }
+
+    import sonic-vlan {
```

Comment:
Hi @shdasari , you may need to comment out vlan import. As it will break libyang backlinks. Issue can be tracked here: https://github.com/sonic-net/sonic-buildimage/issues/9312

Reply:
Sure, will upload a new set of diffs with the changes commented out.